carrier_restrict_connection/models/stock_picking_type.py

```diff
@@ -0,0 +1,10 @@
+# Copyright 2023 Manuel Regidor <manuel.regidor@sygel.es>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class StockPickingType(models.Model):
+    _inherit = "stock.picking.type"
+
+    set_carrier_connection = fields.Boolean()
```

Comment:
It should be indicated that the default output movements are activated.

Reply:
I have added a help @ValentinVinagre

Reply:
@manuelregidor Do you see it correctly like this?

Reply:
@ValentinVinagre I don't understand the problem you see, you can direct message me in if you want

Reply:
There's no problem. I told you that a default option needs to be set, and you've added a help feature. I'll ask Manu how he views this point I mentioned, since he may have it under control and doesn't see the need for it.

Reply:
default = True|False -> **no**.
The default needs to call a function to do the following:
If the "operation type" is "outgoing," it should be enabled by default; otherwise, it will be False.
Make sure it does this in all locations, depending on the multi-warehouse and multi-company configurations.

![Selección_268](https://github.com/user-attachments/assets/728bd568-77bb-4e13-b04f-c8f1dc949f87)

Reply:
@ValentinVinagre I have applied the cange, can you review?